README.md

```diff
@@ -201,6 +201,23 @@ You can configure the storage backend using the `session.session.storageOptions`
 
 Check out here what storage backends are supported and how to configure them: https://github.com/unjs/unstorage#drivers
 
+### IP Pinning
+
+For increased security, you can enable the `ipPinning` flag in the session options. This will make it so sessions are bound by both the session's ID and the user's IP, which means no [session-jacking](https://owasp.org/www-community/attacks/Session_hijacking_attack).
+
+For this feature to work, we rely on [`Socket#remoteAddress`](https://nodejs.org/api/net.html#socketremoteaddress) which means you might have to configure your (reverse) proxy to properly forward IP addresses (Usually via the `X-Forwarded-For` header):
+
+* [Using Apache 2](https://httpd.apache.org/docs/current/mod/mod_proxy.html#x-headers)
+* [Using NGINX](https://www.nginx.com/resources/wiki/start/topics/examples/forwarded/)
+* [Using CloudFlare](https://developers.cloudflare.com/fundamentals/get-started/reference/http-request-headers/)
+
+Supported headers (in order):
+* `X-Forwarded-For`
+* `True-Client-Ip`
+* `CF-Connecting-Ip`
```

Comment:
It would indeed be better to let users configure which header they allow all the way from web-server to web-app.

I think the `Enabled` part would be redundant since it's a boolean flag. I personally don't think having an option enabling a related feature is a good idea. It could be quite confusing. But I get where you're coming from, and I think we could allow `ipPinning` to be either a simple boolean flag, or an object of configuration options (which could include `forwardedIpHeaders`)

Reply:
I meant to suggest two separate keys:
```ts
{
    // ...
    ipPinningEnabled: false,
    ipPinningHeaderName: null,
}
```

But having an object can also work
```ts
{
    // disabled
    ipPinning: null
}
{
    // enabled, but no headers are processed
    ipPinning: true || {}
}
{
    // only specified header is processed, local header is ignored
    ipPinning: {
        headerName: 'X-Forwarded-For'
    }
}
```

At least I can't come up a use-case where specifying `headers` as an array is needed

Reply:
I'll do a single header name for name, it can be trivially changed in the future